enterprise/maven.jaxws/src/org/netbeans/modules/maven/jaxws/MavenModelUtils.java

```diff
@@ -59,14 +58,28 @@
  */
 public final class MavenModelUtils {
 
+    private static Profile profile;
```

Comment:
This will become a problem. If you have one maven project with JakartaEE and one with JavaEE open, both will use the same `profile`. You will need to pass the "isJakarta" property (or the profile) from the external entry points, to decide per call.

A passing question: is it save to assume this is only called for JavaEE/JakarataEE projects? If it is called for JavaSE projects, there will be no profile.